src/config.js

```diff
@@ -1,8 +1,18 @@
-export const firebaseConfig = {
-  apiKey: 'AIzaSyDOTe8SVN_5WGtuKdXQoQy5slhAq3kEIbs',
-  authDomain: 'findy-id.firebaseapp.com',
-  databaseURL: 'https://findy-id.firebaseio.com',
-  projectId: 'findy-id',
-  storageBucket: 'findy-id.appspot.com',
-  messagingSenderId: '943881521886'
-}
+export const firebaseConfig =
+  process.env.NODE_ENV === 'production'
```

Comment:
Respect to NODE_ENV.

We require to specify `NODE_ENV` when build (e.g. in webpack@4, only `--mode production` option) at using side.